controllers/user.controller.js

```diff
@@ -1,44 +1,55 @@
 const DB = require("../configs/db.config");
+const { userValidator } = require("../validators/user.validator");
+const bcrypt = require("bcrypt");
 
 /*
   @desc   Register new user
   @route  POST /api/user/signup
   @access Public
 */
-const signUp = async (req, res) => {
+
+exports.signUp = async (req, res) => {
+  const { error } = await userValidator(req.body);
+  if (error) {
+    return res.status(422).json({ message: error.message });
+  }
   let user = await DB.getUserByEmail(req.body.email);
   if (user) return res.status(400).send("User already registered!");
 
```

Comment:
@ashabhussan bhai, please try to use `process.env` in the existing code. and make a file as a named config.